3rd-party-tools/plink-regenie/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM ubuntu:latest 
+
+LABEL MAINTAINER="Broad Institute DSDE <dsde-engineering@broadinstitute.org>" 
+
+WORKDIR /usr
+
+# need to apt-get everything for python since we can only copy pip installed packages
+RUN set -eux; \
```

Comment:
If we change to python:3.10, we do not need to add wget, etc. it will already be there